### ISSUE TITLE
Memory error while loading wiki-news model for incremental learning

### ISSUE THREAD

**Issue by vdpappu** [ericxsun/fastText#4]
Hi Eric, 
Thanks for the excellent enhancement. I am trying to use your repo for incremental learning. I am getting a memory error while running the script. My machine has 32gb ram and I am able to load the pre-trained model otherwise for inference tasks. 

![image](https://user-images.githubusercontent.com/16121695/44082346-8e87c4e0-9fce-11e8-8147-30ec31700e7e.png)

Pre-trained model size: 6.8gb
Command executed: 
./fasttext skipgram -input /home/aaa/Downloads/datasets/nlu/sed_sof_corpus.txt -inputModel /home/aaa/Downloads/datasets/wiki-news-300d-1M-subword.bin -output sed_sof_trlearn -incr

**Comment by airlsyn** [comment]
that's weird. I have not met such error, even my dataset is larger than your. I'll check it. 

You could try to add more info to locate the line of code which caused such error. Any effort will be appreciated.

**Comment by vdpappu** [comment]
Sure. Will try that out and share the details.

**Comment by vdpappu** [comment]
I am able to run the same with in a Mac. May be some issue with my old machine. However, its not taking the set learning rate. It always shows 0.0000. 

![image](https://user-images.githubusercontent.com/16121695/44520043-7037d500-a6ec-11e8-8e95-38f6be33b31c.png)
